cloudify_cli/commands/logs.py

```diff
@@ -41,22 +41,23 @@ def get_host_date(conn):
     return conn.run('date +%Y%m%dT%H%M%S', hide=True).stdout.strip()
 
 
-def _archive_logs(conn, node_type, logger):
+def _archive_logs(conn, node_type, logger, node_ip=None):
     """Creates an archive of all logs found under /var/log/cloudify plus
     journalctl.
     """
     archive_filename = 'cloudify-{node_type}-logs_{date}_{ip}.tar.gz'.format(
         node_type=node_type,
         date=get_host_date(conn),
-        ip=env.profile.manager_ip
+        ip=node_ip or env.profile.manager_ip
```

Comment:
probably better to just make this always take the ip, then it won't have the env.profile dependency

Reply:
Fixed